### PR TITLE
Made the mod compatible with model-tweaking mods (like Mo' Bends)

### PR DESCRIPTION
**Pull request by iwoplaza**
7de265acb42024e1eeb295cbdf40878e

### CODE REVIEW

File: src/main/java/net/mcft/copy/backpacks/ProxyClient.java [hunk 1]
```diff
@@ -88,8 +88,10 @@ public void init() {
 		
 		RenderManager manager = mc.getRenderManager();
 		Map<String, RenderPlayer> skinMap = manager.getSkinMap();
-		skinMap.get("default").addLayer(new RendererBackpack.Layer());
-		skinMap.get("slim").addLayer(new RendererBackpack.Layer());
+		for (RenderPlayer renderer : skinMap.values())
+		{
```
Comment (asiekierka) [inline src/main/java/net/mcft/copy/backpacks/ProxyClient.java]: Please match the coding style of the mod. We want to keep uniform codestyle.

Comment (iwoplaza) [reply src/main/java/net/mcft/copy/backpacks/ProxyClient.java]: Ah, sorry. Yeah, I kept that in mind in the other files, but this was the first one I did, and I forgot to come back here.

Comment (iwoplaza) [reply src/main/java/net/mcft/copy/backpacks/ProxyClient.java]: Should I change it, or will you just fix the style when potentially merging?

Comment (asiekierka) [reply src/main/java/net/mcft/copy/backpacks/ProxyClient.java]: You should change it. I have other issues to review.

**Pull request merged**